components/encoder/incremental/incremental_test.go

```diff
@@ -56,6 +56,11 @@ func TestEncoder(t *testing.T) {
 	deps := make(resource.Dependencies)
 	deps[board.Named("main")] = b
 
+	i1, ok := b.DigitalInterruptByName("11")
+	test.That(t, ok, test.ShouldBeTrue)
+	i2, ok := b.DigitalInterruptByName("13")
+	test.That(t, ok, test.ShouldBeTrue)
+
```

Comment:
these are going to cause issues because [DigitalInterruptByName returns an error now](https://viam.atlassian.net/browse/RSDK-7328)